### PR TITLE
Remove nailgun.entities imports in tests/foreman/longrun

### PR DESCRIPTION
**Pull request by tpapaioa**
### Problem Statement

SAT-22504

Uses of `nailgun.entities.${ENTITY}` should be replaced with `${SATELLITE_INSTANCE}.api.${ENTITY}`.

### Solution

All tests under `tests/foreman/longrun/` now use nailgun through the satellite instance's `api` attribute, and imports of `nailgun.entities` have been removed.

Test results are the same before and after these changes, and the test failure is unrelated.

PRT:

job/robottelo-pr-testing/7183/console

```
14:59:31  =========================== short test summary info ============================
14:59:31  ERROR tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7] - robottelo.exceptions.CLIReturnCodeError: CLIReturnCodeError(status=70, stderr='Error: scap_content not found.\n', msg='Command "scap-content info" finished with status 70\nstderr contains:\nError: scap_content not found.\n'
14:59:31  ============= 1 passed, 45 warnings, 1 error in 903.10s (0:15:03) ==============
```

PRT against master:

job/robottelo-pr-testing/7213/console

```
09:52:46  =========================== short test summary info ============================
09:52:46  ERROR tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7] - robottelo.exceptions.CLIReturnCodeError: CLIReturnCodeError(status=70, stderr='Error: scap_content not found.\n', msg='Command "scap-content info" finished with status 70\nstderr contains:\nError: scap_content not found.\n'
09:52:46  ============= 1 passed, 46 warnings, 1 error in 1034.34s (0:17:14) =============
```

### CODE REVIEW

**Comment by tpapaioa** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/longrun/test_inc_updates.py::test_positive_noapply_api[rhel7_contenthost_module0] tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7]
```

**Comment by tpapaioa** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/longrun/test_inc_updates.py::test_positive_noapply_api[rhel7_contenthost_module0] tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7]
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7183
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/longrun/test_inc_updates.py::test_positive_noapply_api[rhel7_contenthost_module0] tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7] --external-logging
Test Result : ============= 1 passed, 45 warnings, 1 error in 903.10s (0:15:03) ==============
```

File: tests/foreman/longrun/test_inc_updates.py [hunk 1]
```diff
@@ -158,17 +159,17 @@ def host(
...
+    module_target_sat.api.RPMContentViewFilter(
         content_view=module_cv, inclusion=True, name='Include Nothing'
     ).create()
     module_cv.publish()
     module_cv = module_cv.read()
     return rhel7_contenthost_module
 
 
-def get_applicable_errata(repo):
+def get_applicable_errata(satellite, repo):
```
Comment (vsedmik) [inline tests/foreman/longrun/test_inc_updates.py:170]: We could probably get rid of this helper completely since it's just an one-liner and used just once.
```suggestion
```
I understand this is out of scope of this PR, non-blocking.

**Review by vsedmik — APPROVED** [review]

**Comment by tpapaioa** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/longrun/test_inc_updates.py::test_positive_noapply_api[rhel7_contenthost_module0] tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7]
```

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7232
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/longrun/test_inc_updates.py::test_positive_noapply_api[rhel7_contenthost_module0] tests/foreman/longrun/test_oscap.py::test_positive_oscap_run_via_ansible[rhel7] --external-logging
Test Result : ============= 1 passed, 40 warnings, 1 error in 931.41s (0:15:31) ==============
```

File: tests/foreman/longrun/test_inc_updates.py (hunk 1 above)
Comment (vsedmik) [reply tests/foreman/longrun/test_inc_updates.py]: Thank you @tpapaioa !
